Source/Components/RiveViewModel.swift

```diff
@@ -119,29 +119,64 @@ open class RiveViewModel: NSObject, ObservableObject, RiveFileDelegate, RiveStat
         didSet { riveView?.alignment = alignment }
     }
     
-    open func play(animationName: String? = nil, loop: Loop? = nil) {
+    /// Starts the active Animation or StateMachine from it's last position. It will start
+    /// from the beginning if the active Animation has ended or a new one is provided.
+    /// - Parameters:
+    ///   - animationName: The name of a new Animation to play on the current Artboard
+    ///   - loop: The loop mode for the active Animation
+    open func play(animationName: String? = nil, loop: Loop = .loopAuto, direction: Direction = .directionAuto) {
```

Comment:
So my answer is... maybe?

Reply:
`direction` looks new here, but I feel like I've seen this in the iOS API's at least before at some point.

Agreed on if we really want these params - but I'd pitch if we don't, we add a deprecation message instead of remove for now, and remove it on a major bump to be totally safe.